src/compiler/diagnosticMessages.json

```diff
@@ -2260,6 +2260,10 @@
         "category": "Error",
         "code": 2718
     },
+    "Class '{0}' incorrectly implements class '{1}'. Did you mean 'extends'?": {

```

Comment:
I'd prefer if the phrasing was more a little more specific on what's being discussed. Maybe something more like:

```
Class '{0}' incorrectly implements class '{1}'. Did you mean to extend '{1}' and inherit its members as a subclass?
```